cfgov/unprocessed/css/molecules/nav-link.less

```diff
@@ -0,0 +1,80 @@
+/* =========================================================================
+   cfgov-refresh
+   m-nav-link
+   ========================================================================= */
+
+/* topdoc
+  name: Nav Link
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <a class="m-nav-link"
+           href="#">
+        </a>
+      codenotes:
+        - ".m-nav-link"
+    - name: Parent link modifier
+      markup: |
+        <a class="m-nav-link
+                  m-nav-link__parent"
+           href="#">
+           Parent nav link
+        </a>
+      codenotes:
+        - ".m-nav-link.m-nav-link__parent"
+      notes:
+        - "Parent links are styled as heading-4."
+    - name: Current link modifier
+      markup: |
+        <a class="m-nav-link
+                  m-nav-link__current"
+           href="#">
+           Current nav link
+        </a>
+      codenotes:
+        - ".m-nav-link.m-nav-link__current"
+      notes:
+        - "Current links have a left green border on all screens."
+  tags:
+    - cfgov-molecules
+*/
+
+.m-nav-link {
+    .u-inline-block();
```

Comment:
I'm not sure about the auto-generation, but `.u-inline-block()` isn't different than `display: inline-block;` except for on IE7 and below, which we can transition away from supporting https://github.com/cfpb/cf-core/blob/gh-pages/src/cf-utilities.less#L120